shm/inc/ship.hpp

```diff
@@ -21,8 +23,15 @@ class Ship {
     size_t getSpeed() const { return speed_; }
     std::string getName() const { return name_; }
     size_t getId() const { return id_; }
+    Cargo getCargo(size_t index) const{
+        if(index <  0){

```

Comment:
```suggestion
        if (index < 0) {

```
I might be wrong but I guess it should be formatted this way.